src/components/dialog/Dialog.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState, useEffect} from "react";
+import "./Dialog.scss";
+import {createPortal} from "react-dom";
+import FocusTrap from "focus-trap-react";
+
+const Dialog = ({title, children, showModal}) => {
+    const [modalVisible, setModalVisible] = useState(showModal);
+    useEffect(() => {
+        setModalVisible(showModal);
+    }, [showModal])
+
+    return modalVisible

```

Comment:
what changes if we rewrite this line to `return showModal` ?

Reply:
Visibility is controlled, inside via modalVisible, and outside by showModal. However, modalVisible is initialized with show modal.